client/components/push-notification/prompt.jsx

```diff
@@ -79,13 +80,25 @@ const PushNotificationPrompt = React.createClass( {
 		return <Notice className="push-notification__notice" text={ noticeText } icon="bell" onDismissClick={ this.props.dismissNotice } />;
 	},
 
+	isUnsupportedChromeVersion: function() {
```

Comment:
Absolutely. I'm still testing it, and the easiest was to push something so that I can test it from VMs without proxying, hence the duplication. Thanks for the advice on where to extract the logic!